src/main/java/org/tkit/onecx/announcement/domain/daos/AnnouncementDAO.java

```diff
@@ -48,6 +49,11 @@ public PageResult<Announcement> loadAnnouncementByCriteria(AnnouncementSearchCri
             cq.select(root).distinct(true);
 
             List<Predicate> predicates = new ArrayList<>();
+            if (criteria.getTitle() != null && !criteria.getTitle().isBlank()) {
+                predicates.add(
+                        QueryCriteriaUtil.createSearchStringPredicate(cb, root.get(Announcement_.TITLE),
+                                criteria.getTitle().concat("*")));
```

Comment:
1. Please check source code of the `QueryCriteriaUtil`
2. Why  why do we need concat for title?

Reply:
pls. use this pattern:

predicates.add(
                        cb.like(...Table.get(TITLE),
                                QueryCriteriaUtil.wildcard(criteria.getTitle(), false)));